is/simple/deployment.yaml

```diff
@@ -0,0 +1,363 @@
+            </Mapping>
+            <Mapping>
+                <Actions>
+                    <Action>tools</Action>
+                </Actions>
+                <Class>org.wso2.carbon.ui.transports.fileupload.ToolsFileUploadExecutor</Class>
+            </Mapping>
+            <Mapping>
+                <Actions>
+                    <Action>toolsAny</Action>
+                </Actions>
+                <Class>org.wso2.carbon.ui.transports.fileupload.ToolsAnyFileUploadExecutor</Class>
+            </Mapping>
+        </FileUploadConfig>
+
+
+        <HttpGetRequestProcessors>
+            <Processor>
+                <Item>info</Item>
+                <Class>org.wso2.carbon.core.transports.util.InfoProcessor</Class>
+            </Processor>
+            <Processor>
+                <Item>wsdl</Item>
+                <Class>org.wso2.carbon.core.transports.util.Wsdl11Processor</Class>
+            </Processor>
+            <Processor>
+                <Item>wsdl2</Item>
+                <Class>org.wso2.carbon.core.transports.util.Wsdl20Processor</Class>
+            </Processor>
+            <Processor>
+                <Item>xsd</Item>
+                <Class>org.wso2.carbon.core.transports.util.XsdProcessor</Class>
+            </Processor>
+        </HttpGetRequestProcessors>
+
+        <DeploymentSynchronizer>
+            <Enabled>false</Enabled>
+            <AutoCommit>false</AutoCommit>
+            <AutoCheckout>true</AutoCheckout>
+            <RepositoryType>svn</RepositoryType>
+            <SvnUrl>http://svnrepo.example.com/repos/</SvnUrl>
+            <SvnUser>username</SvnUser>
+            <SvnPassword>password</SvnPassword>
+            <SvnUrlAppendTenantId>true</SvnUrlAppendTenantId>
+        </DeploymentSynchronizer>
+
+        <ServerInitializers>
+
+        </ServerInitializers>
+
+        <RequireCarbonServlet>${require.carbon.servlet}</RequireCarbonServlet>
+
+        <StatisticsReporterDisabled>true</StatisticsReporterDisabled>
+
+        <FeatureRepository>
+          <RepositoryName>default repository</RepositoryName>
+          <RepositoryURL>http://product-dist.wso2.com/p2/carbon/releases/wilkes/</RepositoryURL>
+        </FeatureRepository>
+
+       <APIManagement>
+
+      <Enabled>true</Enabled>
+
+      <LoadAPIContextsInServerStartup>true</LoadAPIContextsInServerStartup>
+       </APIManagement>
+    </Server>
+kind: ConfigMap
+metadata:
+  name: identity-server-conf
+  namespace: wso2
+
+---
+# Deploying kubernetes services
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2is-service
+  namespace: wso2
+spec:
+  selector:
+    deployment: wso2is
+  ports:
+    - name: servlet-http
+      port: 9763
+      targetPort: 9763
+      protocol: TCP
+    - name: servlet-https
+      port: 9443
+      targetPort: 9443
+      protocol: TCP
+
+#SLEEP 10S
+
+---
+
+# Deploying wso2 identity server
+
+apiVersion: apps/v1beta1
```

Comment:
@NishikaDeSilva it is better if we move to the new Kubernetes API version for [Deployment](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#deployment-v1-apps)s.

Reply:
done and committed for is